app/models/case_log.rb

```diff
@@ -439,6 +439,91 @@ def created_by_name
     created_by&.name
   end
 
+  def scheme_code
+    scheme&.id ? "S#{scheme.id}" : nil
+  end
+
+  def scheme_name
```

Comment:
Could use delegate for these instead perhaps? https://apidock.com/rails/Module/delegate